brytescore/src/main/java/com/brytecore/Brytescore.java

```diff
@@ -127,10 +127,31 @@ public Brytescore(Context context, String apiKey) {
         // Generate and save unique session ID
         sessionId = generateUUID();
         editor.putString("brytescore_session_sid", sessionId);
-        editor.apply();
 
         // Retrieve user ID from brytescore_uu_uid
-        userId = preferences.getInt("brytescore_uu_uid", -1);
+        Map<String, ?> all = preferences.getAll();
```

Comment:
@mcgomez consider adding a comment here to note that we handle ints for backwards compatibility, but user ids should be a string